src/test/java/org/folio/circulation/services/ItemsInTransitReportServiceTest.java

```diff
@@ -1,23 +1,30 @@
 package org.folio.circulation.services;
 
-import static org.junit.jupiter.api.Assertions.assertNotNull;
-
-import java.util.concurrent.CompletableFuture;
-
+import io.vertx.core.json.JsonObject;
+import org.folio.circulation.support.GetManyRecordsClient;
 import org.folio.circulation.support.results.Result;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
 
-import io.vertx.core.json.JsonObject;
+import java.util.concurrent.CompletableFuture;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
```

Comment:
according to convention, static imports' orders should come before non-static imports

Reply:
thanks